proto/zitadel/user/v2/user.proto

```diff
@@ -276,13 +276,60 @@ message Passkey {
   ];
 }
 
+message AuthFactor {
+  AuthFactorState state = 1 [
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      description: "current state of the auth factor";
+    }
+  ];
+  oneof type {
+    AuthFactorOTP otp = 2 [
+      (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+        description: "one type use OTP, OTPSMS, OTPEmail or U2F"
```

Comment:
Please use a fitting description for the type of the factor, even if the same description is used by the old endpoint.